lib/openstax/aws/stack.rb

```diff
@@ -361,10 +361,52 @@ def exists?
       "DOES_NOT_EXIST" != status
     end
 
+    def all_statuses
+      %w(
+        CREATE_IN_PROGRESS
+        CREATE_FAILED
+        CREATE_COMPLETE
+        ROLLBACK_IN_PROGRESS
+        ROLLBACK_FAILED
+        ROLLBACK_COMPLETE
+        DELETE_IN_PROGRESS
+        DELETE_FAILED
+        DELETE_COMPLETE
+        UPDATE_IN_PROGRESS
+        UPDATE_COMPLETE_CLEANUP_IN_PROGRESS
+        UPDATE_COMPLETE
+        UPDATE_ROLLBACK_IN_PROGRESS
+        UPDATE_ROLLBACK_FAILED
+        UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS
+        UPDATE_ROLLBACK_COMPLETE
+        REVIEW_IN_PROGRESS
+        IMPORT_IN_PROGRESS
+        IMPORT_COMPLETE
+        IMPORT_ROLLBACK_IN_PROGRESS
+        IMPORT_ROLLBACK_FAILED
+        IMPORT_ROLLBACK_COMPLETE
+      )
+    end
+
+    def active_statuses
+      all_statuses - %w(CREATE_FAILED DELETE_COMPLETE)
+    end
+
     def defines_secrets?
       !secrets_blocks.empty?
     end
 
+    def self.query(regex: /.*/, regions: %w(us-east-1 us-east-2 us-west-1 us-west-2), active: true)
```

Comment:
You are not using `regex` in this method

Reply:
Lol

Reply:
Fixed this.